### ISSUE TITLE
nodejs.10x runtime not supported for lambda functions

### ISSUE THREAD

**Issue by dwaynejb94** [terraform-providers/terraform-provider-aws#8658]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.11.13
+ provider.aws v1.60.0
+ provider.consul v1.1.0

### Affected Resource(s)

* aws_lambda_function

```hcl
resource "aws_lambda_function" "event-collector" {
  function_name = "namer"
  description = "description"
  handler = "index.handler"
  role = "${role.arn}"
  runtime = "nodejs10.x"
  tags = "${var.tags}"
  timeout = 300
  publish = false
  s3_bucket = "${buckey}"
  s3_key = "key"
}

### Expected Behavior

nodejs.10x is now supported for lambda so the resource should be created.

### Actual Behavior

Error: module.data-store.aws_lambda_function.event-collector: expected runtime to be one of [dotnetcore1.0 dotnetcore2.0 dotnetcore2.1 go1.x java8 nodejs4.3 nodejs4.3-edge nodejs6.10 nodejs8.10 provided python2.7 python3.6 python3.7 ruby2.5], got nodejs10.x

### Steps to Reproduce

terraform apply on a aws_lambda_function resource with 'nodejs.x' set as the runtime

**Comment by bflad** [comment]
Hi @dwaynejb94  👋 Thank you for this feature request. This support was completed as part of #8627 and #8622. It will be released in version 2.11.0 of the Terraform AWS Provider, likely later today. 👍

**Comment by rpstreef** [comment]
Has it been added to aws_lambda_layer_version as well?

**Comment by bflad** [comment]
This has been released in [version 2.11.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2110-may-17-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

> Has it been added to aws_lambda_layer_version as well?

Not that I'm aware of -- please submit a new feature request and we can get that in.

**Comment by rpstreef** [comment]
@bflad you should probably add to the release notes that this also applies to aws_lambda_layer_version.

Tested this and it works
